actionview/lib/action_view/helpers/form_options_helper.rb

```diff
@@ -897,16 +897,16 @@ def weekday_select(method, options = {}, html_options = {})
         @template.weekday_select(@object_name, method, objectify_options(options), @default_html_options.merge(html_options))
       end
 
-      # Wraps ActionView::Helpers::FormOptionsHelper#collection_check_boxes for form builders:
+      # Wraps ActionView::Helpers::FormOptionsHelper#collection_checkboxes for form builders:
       #
       #   <%= form_for @post do |f| %>
-      #     <%= f.collection_check_boxes :author_ids, Author.all, :id, :name_with_initial %>
+      #     <%= f.collection_checkboxes :author_ids, Author.all, :id, :name_with_initial %>
       #     <%= f.submit %>
       #   <% end %>
       #
       # Please refer to the documentation of the base helper for details.
-      def collection_check_boxes(method, collection, value_method, text_method, options = {}, html_options = {}, &block)
-        @template.collection_check_boxes(@object_name, method, collection, value_method, text_method, objectify_options(options), @default_html_options.merge(html_options), &block)
+      def collection_checkboxes(method, collection, value_method, text_method, options = {}, html_options = {}, &block)
```

Comment:
hi @byroot I think you forgot to add an alias here as well as in line 782 
(sorry about the screenshot without context but i could reproduce it on our prod app ) 
<img width="691" alt="image" src="https://github.com/user-attachments/assets/8240742c-61f1-404d-9e5a-0b62f322c845">

Reply:
I opened https://github.com/rails/rails/pull/52436